cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -52,15 +51,23 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/admission/securitycontext/scdeny"
 	"k8s.io/kubernetes/plugin/pkg/admission/serviceaccount"
 	"k8s.io/kubernetes/plugin/pkg/admission/storageclass/setdefault"
+
+	"k8s.io/apiserver/pkg/admission/plugin/initialization"
+	"k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle"
+	mutatingwebhook "k8s.io/apiserver/pkg/admission/plugin/webhook/mutating"
+	validatingwebhook "k8s.io/apiserver/pkg/admission/plugin/webhook/validating"
+	genericoptions "k8s.io/apiserver/pkg/server/options"
 )
 
-// RegisterAllAdmissionPlugins registers all admission plugins
-func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
-	admit.Register(plugins)
+// RegisterAllAdmissionPlugins registers all admission plugins and
+// sets the recommended plugins order.
+func RegisterAllAdmissionPlugins(admission *genericoptions.AdmissionOptions) {
```

Comment:
I think this is two separate functions.  One that registers all the plugins into a plugin registry.  Then a second function that uses that function and applies defaults for flags.

Reply:
ok